egs/aishell/s10/chain/feat_dataset.py

```diff
@@ -55,21 +56,36 @@ def _add_model_left_right_context(x, left_context, right_context):
 
 class FeatDataset(Dataset):
 
-    def __init__(self, feats_scp):
+    def __init__(self, feats_scp, ivector_scp=None):
         assert os.path.isfile(feats_scp)
 
         self.feats_scp = feats_scp
 
-        # items is a list of [key, rxfilename]
-        items = list()
+        # items is a dict of {key: [key, rxfilename, ivec]}
```

Comment:
`[key, feat_rxfilename,e ivec_rxfilename]` is better.

Reply:
this data_dict form is from #3947 , self.items will be ites.values(), Should I change the comment ?

Reply:
yes, please.
`rxfilename` was used here because there was only `feats.scp`.
Now ivector is added, so `rxfilename` is a confusing name here.